src/scss/common/registration.scss

```diff
@@ -12,3 +12,15 @@
     display: block;
   }
 }
+
+.password-group {
```

Comment:
I think it may be worth moving these somewhere else, since they're not really only used during registration.